content/tools.md

```diff
@@ -11,13 +11,16 @@ toc: true
 - [Google Maps API](https://developers.google.com/maps/get-started/)
 - [Interactive periodic table of the elements](https://leonard-seydoux.github.io/periodic-table/)
 - [Periodic Table of Elements](https://www.ptable.com)
+- [3D earthquake display](https://glowy-earthquakes.glitch.me)
 
 ### IRIS
```

Comment:
Not sure if it's a good idea to put the IRIS tools here.

Reply:
Let's keep it for a while.

Reply:
I update the categories in 8350fb6. See if you feel okay